data/mods/blazemod/modinfo.json

```diff
@@ -4,10 +4,9 @@
     "ident": "blazemod",
```

Comment:
Not sure if I'm jumping the gun here on the maintainer field, but I've reversed my wrong correction of Blaze's handle as per Brian's suggestion.

Reply:
This is an array and you can honor the original creator too.